### DISCUSSION TITLE
Add ARM64 images

### DISCUSSION THREAD

**Discussion by tibeer** [inovex/scrumlr.io#3994] [Proposals]
Hi!

I would like to ask if arm64 images are on the roadmap. I took a look package github pipeline and it should be easy to add multi-arch support during the docker/build-push-action@v4 actions, I can even create a PR for this, but I am unsure, how to deal with the tag names in the future or if you would like to switch to multi-arch tags.

Kind regards
Tim

**Comment by Schwehn42** [comment]
Hi Tim, thanks for your idea. To be honest, I am not familiar with the technical details, so @Lennart01 could you take a look at this?

**Comment by Lennart01** [comment]
Since we already planned on switching our backend build chain to `ko` I think going for multi-arch images here would be a nice addition.
Honestly idk why we never built ARM images :D.
Im confident we will get this into the next release 👍

**Reply by Lennart01** [reply]
Addressed in #4009.
Since we currently can't get a hold of ARM runners we will only build ARM images for releases.
If you're interested in dev versions (for whatever reason) you still need to build them urself until ARM runners go public :).

**Reply by tibeer** [reply]
Sorry for the late reply: ARM images for releases would already help a lot :) Thanks for your help!

**Comment by Lennart01** [comment]
@tibeer 
We just released version 3.8.1 with support for `linux/arm64`
🥳

**Reply by tibeer** [reply]
Absolutely awesome! Will try it tomorrow =))

**Reply by tibeer** [reply]
@Lennart01 bad news - the image for the backend gives me an `exec /app/main: exec format error`. I tripple checked by first pulling the image by the sha and then removing all stale chaced images and trying again. After that I pulled the image locally and started it with

```
docker run -it --rm --entrypoint sh ghcr.io/inovex/scrumlr.io/scrumlr-server@sha256:e3bb0f179e6b61cfed4679c5340d2a9a10c8ce55389fb435ef817d5692fff79a
```

After that, I installed `file` to check the contained binary:

```
/app # apk add file
fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/main/aarch64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/community/aarch64/APKINDEX.tar.gz
(1/2) Installing libmagic (5.45-r1)
(2/2) Installing file (5.45-r1)
Executing busybox-1.36.1-r29.trigger
OK: 17 MiB in 16 packages
/app # file main
main: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=6EpIJFWZmLqGhd_MUPdt/HIHL_dlucA0S4P481dXw/O4zzLBc6NN7XXPzEUlTC/Q1wy6xTq0gCzt6ljwgCU, with debug_info, not stripped
/app #
```

It looks like the go binary is still an x86-64 binary :/

**Reply by tibeer** [reply]
I tried to create a PR to tackle this: #4361

**Reply by Lennart01** [reply]
Woops sorry. Ill take a look :D.

**Reply by Lennart01** [reply]
@tibeer 
We released 3.8.2 to address this issue.

**Reply by tibeer** [reply]
Tested it, works perfectly! Thank you so much :)
